src/components/calcite-card/calcite-card.tsx

```diff
@@ -0,0 +1,186 @@
+import { Component, Element, Host, Method, Prop, h } from "@stencil/core";
+
+import { getElementDir } from "../../utils/dom";
+import { CSS_UTILITY } from "../../utils/resources";
+import { VNode } from "@stencil/core/dist/declarations";
+
+import { CSS, SLOTS } from "./resources";
+import CalciteScrim from "../../utils/CalciteScrim";
+import classnames from "classnames";
+
+/**
+ * @slot thumbnail - [Required] A slot for adding a thumnail to the card.
+ * @slot header - [Required] A slot for adding a heading and an icon to the card.
+ * @slot action - A slot for adding a single action as a button.
+ * @slot footer-leading - A slot for adding a leading footer.
+ * @slot footer-trailing - A slot for adding a trailing footer.
+ */
+
+@Component({
+  tag: "calcite-card",
+  styleUrl: "./calcite-card.scss",
+  shadow: true
+})
+export class CalciteCard {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * When true, the card can't be clicked and is visually muted.
+   */
+  @Prop({ reflect: true }) disabled = false;
+
+  /**
+   * When true, the cards content is waiting to be loaded. This state shows a busy indicator.
+   */
+  @Prop({ reflect: true }) loading = false;
+
+  /**
+   * Indicates whether the image's height or width is prioritized.
+   * When false, the image fills the whole thumbnail space.
+   * When true, the height of the thumbnail is used to crop.
+   */
+  @Prop({ reflect: true }) imgHeightPriority = false;
```

Comment:
Maybe `respectImageHeight` or `respectImageWidth` would be better for a name?

Reply:
or `fillHeight/fillWidth`?

Reply:
RESPEKT